src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,61 @@ Keywords:
 
 # Release notes
 
+## March 7, 2024
+
+### Roku SDK 1.1.0
+
+- Streaming Media for Edge Network
+    - Added new APIs and configuration for tracking media.
+    - Customizable ping interval.
+- Added support to create multiple instances of the SDK.
+- Support non-xdm data in `SendEvent` API
+- A comprehensive [migration guide](https://github.com/adobe/aepsdk-roku/blob/main/Documentation/media-migration.md) to transition from [Adobe Media SDK](https://github.com/Adobe-Marketing-Cloud/media-sdks/tree/master/sdks/roku) to [Adobe Experience Platform Roku SDK](https://github.com/adobe/aepsdk-roku).
+
+### iOS Assurance 4.1.2
+
+* Fix an issue where dependency versions did not have an upper bound restriction.
+
+### iOS Analytics 4.0.1
+
+- Added max versions to Platform dependencies in podspec.
+
+### iOS EdgeConsent 4.0.1
+
+- Added max versions to Platform dependencies in podspec.
```

Comment:
@yangyansong-adbe what timezone do we use for determining the date for release notes? Asking because Analytics and EdgeConsent were also released on March 6th with the others.